recipes-bsp/acpica/acpica_20180810.bb

```diff
@@ -16,7 +16,7 @@ COMPATIBLE_HOST = "(i.86|x86_64|arm|aarch64).*-linux"
 
 DEPENDS = "bison flex"
 
-SRC_URI = "https://acpica.org/sites/acpica/files/acpica-unix2-${PV}.tar.gz \
+SRC_URI = "https://www.coreboot.org/releases/crossgcc-sources/acpica-unix2-${PV}.tar.gz \
```

Comment:
Can you please change this to https://downloadmirror.intel.com/774775/acpica-unix2-${PV}.tar.gz

Openembedded has gone this way as well since micledore see https://git.openembedded.org/openembedded-core/tree/meta/recipes-extended/acpica/acpica_20230628.bb?h=nanbield

Note to self: for kirkstone the OE recipe is newer and will be used over this one, however at present they have not yet backported this change.